integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature

```diff
@@ -174,11 +171,77 @@ Feature: PublicLightingScheduleManagement Set Light Schedule
       | InnerMessage | <Message>        |
 
     Examples: 
-      | WeekDay     | ActionTime   | Time         | TriggerType   | Message                                                                                                      |
-      | ABSOLUTEDAY | ABSOLUTETIME | 18:00:00.000 |               | Validation Exception, violations: startDay may not be null when weekDay is set to ABSOLUTEDAY;               |
+      | WeekDay     | ActionTime   | Time         | TriggerType   | Message                                                                                                                                       |
+      | ABSOLUTEDAY | ABSOLUTETIME | 18:00:00.000 |               | Validation Exception, violations: startDay may not be null when weekDay is set to ABSOLUTEDAY;                                                |
       | MONDAY      | SUNRISE      |              | LIGHT_TRIGGER | Validation Exception, violations: triggerWindow may not be null when actionTime is set to SUNRISE or SUNSET and triggerType is LIGHT_TRIGGER; |
       | MONDAY      | SUNSET       |              | LIGHT_TRIGGER | Validation Exception, violations: triggerWindow may not be null when actionTime is set to SUNRISE or SUNSET and triggerType is LIGHT_TRIGGER; |
 
+  Scenario Outline: Set light schedule with for inactive/unregistered device (device lifecycle state)
```

Comment:
"with for" => "for"
Also change this for all the other new scenarios.

Reply:
Done